service/src/intTest/java/uk/nhs/adaptors/gp2gp/gpc/GetGpcDocumentComponentTest.java

```diff
@@ -40,6 +40,7 @@
     private static final String INVALID_DOCUMENT_ID = "non-existing-id";
     private static final String NO_RECORD_FOUND = "NO_RECORD_FOUND";
     private static final String NO_RECORD_FOUND_STRING = "No Record Found";
+    private static final String ODS_CODE_TOKEN = "{ODS_CODE}";
```

Comment:
I'd rename these constants from `ODS_CODE_TOKEN` to `ODS_CODE_PLACEHOLDER` ("TOKEN" to me makes it sound like this is a proper value, but it's a placeholder for a value)